### ISSUE TITLE
Figure outliers

### ISSUE THREAD

**Issue by cglosser** [cgMedia/rabi_paper#9]
> About the intriguing isolated outliers in Fig 3+5: can the authors extend their explanation?

**Comment by cglosser** [comment]
@piermaro Looking over everything, I'm still a little unsure if we've addressed this properly or not. Do you think there's anything else important to say regarding this?

**Comment by piermaro** [comment]
I can't find the "this" on github all issues are closed....

C



Carlo Piermarocchi
Professor of Physics
Department of Physics and Astronomy
Michigan State University
Biomedical Physical Sciences Bldg.
567 Wilson Road, Room 4263
East Lansing, MI 48824
Tel (517) 884-5631
Fax (517) 353 4500
email: piermaro@msu.edu
http://www.pa.msu.edu/~carlo
________________________________

**Comment by piermaro** [comment]
?sorry now I see it...



Carlo Piermarocchi
Professor of Physics
Department of Physics and Astronomy
Michigan State University
Biomedical Physical Sciences Bldg.
567 Wilson Road, Room 4263
East Lansing, MI 48824
Tel (517) 884-5631
Fax (517) 353 4500
email: piermaro@msu.edu
http://www.pa.msu.edu/~carlo
________________________________

**Comment by piermaro** [comment]
I think we explain it in the introduction:" We interpret this as the consequence of a time-dependent en- ergy shift that brings the pair temporarily out of resonance with the external driving field." Maybe this should be repeated in sec IV A at the end of the first paragraph ( maybe rephrased ).
